src/Datasource/EntityTrait.php

```diff
@@ -375,6 +375,87 @@ public function has($property)
     }
 
     /**
+     * Returns whether this entity contains a property named $property
+     * that contains a non-null value.
+     *
+     * ### Example:
+     *
+     * ```
+     * $entity = new Entity(['id' => 1, 'name' => null]);
+     * $entity->has('id'); // true
+     * $entity->has('name'); // false
+     * $entity->has('last_name'); // false
+     * ```
+     *
+     * You can check multiple properties by passing an array:
+     *
+     * ```
+     * $entity->has(['name', 'last_name']);
+     * ```
+     *
+     * All properties must not be null to get a truthy result.
+     *
+     * When checking multiple properties. All properties must not be null
+     * in order for true to be returned.
+     *
+     * @param string|array $property The property or properties to check.
+     * @return bool
+     */
+    public function has($property)
```

Comment:
Is this method deprecated?

Reply:
I decided to keep `has` over the rename. I don't think the rename adds substantial value, and it would be yet another breaking change we'd be adding in for 4.0.